sdk/core/System.ClientModel/src/Options/LoggingOptions.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
```

Comment:
Linq calls tend to be bad for perf - is there another way to do what we're using this for?